portfolio/src/main/java/com/google/sps/validators/TextValidator.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.validators;
+
+public interface TextValidator {
+  // Allow only alphanumeric chars and few extra symbols.
+  public static final String validPattern = "^([A-z]|[0-9]|[\\?!,.(\" ])*$";
```

Comment:
All fields in an interface are public, static, and final. So you don't really need to specify these qualifiers.

All methods in an interface are public, so here too, the qualifiers are redundant.

There's no harm specifying them, but I omit them for they don't add extra information.